Семинар_7q/Task_7-1.py

```diff
@@ -0,0 +1,12 @@
+stix = list(input().split())
+glasnie = {1: "АОУЫЭЕЁИЮЯаоуыэеёиюя"}
+summ = 0
+for i in range(0, len(stix)):
```

Comment:
Во-первых, ноль передавать совершенно необязательно. 
```suggestion
for i in range(len(stix)):
```
Во-вторых, можно проходиться напрямую по фразам списка.
```suggestion
for i in stix:
```